plugins/woocommerce/client/legacy/js/frontend/cart.js

```diff
@@ -268,8 +268,19 @@ jQuery( function ( $ ) {
 		/**
 		 * Toggle Shipping Calculator panel
 		 */
-		toggle_shipping: function () {
-			$( '.shipping-calculator-form' ).slideToggle( 'slow' );
+		toggle_shipping: function ( event ) {
+			var $target = $( event.currentTarget );
+
+			$( '.shipping-calculator-form' ).slideToggle( 'slow', function () {
+				var self = this;
+
+				setTimeout( function () {
```

Comment:
When the jQuery `slideToggle` callback runs, the HTML attributes of the `.shipping-calculator-form` element are not 100% updated. The `setTimeout` function ensures that all DOM manipulation made by the slide effect is done, so it's safe to check whether the form is visible or not.